src/screens/Game/GamePage/index.tsx

```diff
@@ -422,12 +422,12 @@ export default function GamePage(): JSX.Element | null {
             </div>
           </>
         ) : (
-          <UpdateComponent />
+          <UpdateComponent message={t('loading.default')} />
```

Comment:
Perhaps we could use the default message directly into the component in case `message` is undefined. And make the message a optional prop (I believe might be already)

Reply:
My bad, it's actually is possible. Commited the change